pkg/apis/provisioning.cattle.io/v1/rke.go

```diff
@@ -29,6 +29,7 @@ type RKEMachinePool struct {
 	UnhealthyRange               *string                      `json:"unhealthyRange,omitempty"`
 	MachineOS                    string                       `json:"machineOS,omitempty"`
 	DynamicSchemaSpec            string                       `json:"dynamicSchemaSpec,omitempty"`
+	HostnameLengthLimit          int                          `json:"hostnameLengthLimit"`
```

Comment:
Why do we not add the keyword `omitempty` for this field?

Reply:
good catch